lib/mr_bump/slack.rb

```diff
@@ -0,0 +1,50 @@
+require 'slack-notifier'
+require 'pry'
+module MrBump
+  # This class uses a slack webhook to push notifications to slack
+  class Slack
+
+    def initialize(opts)
+      @webhook = opts["webhook_url"]
+      @username = opts["username"]
+      @icon = opts["icon"]
+    end
+
+    def bump(version, changes)
+      options = {}
+      options[:icon_url] = @icon
+      options[:attachments] = [attatchment(version, changes)]
+      puts 'notifying slack'
+      notifier.ping "Mr Bump: new release!", options
```

Comment:
I think this should just be a blank string, its fairly obvious that its Mr Bump and that its a new release anyway